troubleshooting/tools/invalid-utf-8/invalid.py

```diff
@@ -0,0 +1,33 @@
+# small script that writes byte values to a file
+# which if interpreted as utf-8 will be invalid chars
+# can test FLB processing of invalid unicode
+
+myfile = open('log.log', 'ab')
```

Comment:
Consider rename it to invalid_output_file

Reply:
I'll make it configurable via the first arg on command line